trust-quorum/src/coordinator_state.rs

```diff
@@ -137,39 +145,76 @@ impl CoordinatorState {
     // This method is "in progress" - allow unused parameters for now
     #[allow(unused)]
```

Comment:
```suggestion
    #[expect(unused)]
```

Reply:
Done in [50ebf64](https://github.com/oxidecomputer/omicron/pull/8000/commits/50ebf6479db5b3e8e345394a1bcf2943306edf78)